packages/core/src/textures/Texture.ts

```diff
@@ -10,6 +10,8 @@ import { TextureMatrix } from './TextureMatrix';
 
 const DEFAULT_UVS = new TextureUvs();
 
+export type TextureSource = string|HTMLImageElement|HTMLCanvasElement|HTMLVideoElement|BaseTexture;
```

Comment:
you can use `ImageSource` from `@pixi/core` here instead of `HTMLxxx` stuff

Reply:
Btw, we have to test if `Sprite.from(resource)` works too

Reply:
Ok I've updated the type. I've not really written any tests, is that something you can help me with?

Reply:
I dont mean to write test, just to check if it works in pixi-playground. I can do it tomorrow

Reply:
Never mind, its part of our discussion about texture resource parameters.